playbooks/nfs.yml

```diff
@@ -8,3 +8,4 @@
   become: yes
   roles:
     - { role: nfs, nfs_is_client: yes }
+    - cachefilesd
```

Comment:
Can we not add this here? Cachefilesd can be really buggy and I rarely use it, but use NFS all the time.

Reply:
Yep, can do.